### PR TITLE
Update mldesigner component env style

### PR DESCRIPTION
**Pull request by zhengfeiwang**
# Description

Update code that set env for mldesigner defined command component in preferred style.

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/conda.yaml [hunk 1]
```diff
@@ -1,11 +1,14 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
+image: mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04
```
Comment (wangchao1230) [inline sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/conda.yaml:4]: rename?

File: sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/components.py [hunk 2]
```diff
@@ -79,7 +72,7 @@ def score_data(
     (Path(score_output) / "score").write_text("scored with {}".format(model))
 
 
-@command_component(display_name="Evaluate", environment=conda_env)
+@command_component(display_name="Evaluate", environment="./conda.yaml")
```
Comment (0mza987) [inline sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/components.py:75]: rename to env.yaml like the previous pr?

File: sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/conda.yaml [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
 name: project_environment
```
Comment (wangchao1230) [inline sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/conda.yaml:5]: may give this a better env name?

**Review by 0mza987 — APPROVED** [review]

File: sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/components.py (hunk 2 above)
Comment (zhengfeiwang) [reply sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/components.py]: renamed

File: sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/conda.yaml (hunk 1 above)
Comment (zhengfeiwang) [reply sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/conda.yaml]: renamed

File: sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/conda.yaml (hunk 3 above)
Comment (zhengfeiwang) [reply sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/conda.yaml]: named it as `mldesigner_environment`

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**